packages/website/ts/components/top_bar/provider_display.tsx

```diff
@@ -62,7 +61,7 @@ export class ProviderDisplay extends React.Component<ProviderDisplayProps, Provi
             this.props.providerType === ProviderType.Injected ? injectedProviderName : 'Ledger Nano S';
         const isProviderMetamask = providerTitle === constants.PROVIDER_NAME_METAMASK;
         const hoverActiveNode = (
-            <div className="flex items-center p1" style={styles.root}>
+            <Island className="flex items-center p1" style={styles.root}>
```

Comment:
I forget why I added the `style` prop to `Island`, but I think it would be best if we tried not to use it. I guess I'm not entirely sure if these components are `Island`s but, assuming they are: can we get rid of the style injection by adding a `borderRadius` prop to Island, and achieving all the other things by adding a `Container` with padding wrapping the children (and wrapping the `Island` for paddingTop)

Reply:
Ok I played around with removing the style prop from Island but it was messing up some layout down the line. I'll add a tech debt ticket for removing style